samples/corda-simple-application/workflows-kotlin/src/main/kotlin/com/example/flow/SimpleFlow.kt

```diff
@@ -237,20 +238,43 @@ class DeleteState(val key: String) : FlowLogic<SignedTransaction>() {
  * @property key the key for the [SimpleState] to be retrieved.
  */
 @StartableByRPC
-class GetStateByKey(val key: String) : FlowLogic<ByteArray>() {
+class GetStateByKey(val key: String) : FlowLogic<List<SimpleState>>() {
```

Comment:
This should return ByteArray only, as this function is called by remote network as well.
If you want to convert ByteArray into SimpleState, I suggest doing that in corda-client.

Reply:
Ok @sanvenDev , I have retained this as `ByteArray` and converted into `String` in the `corda-client`.